### PR TITLE
Add mask input samples

### LINKED ISSUE #255: Create samples for Mask input component
Create samples for Mask input web component

### PR DESCRIPTION
**Pull request by onlyexeption**
Closes #255

### CODE REVIEW

File: samples/inputs/mask-input/value-modes/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "build:prod:legacy": "webpack --env.NODE_ENV=production --env.legacy=true  --mode production --config ./webpack.config.js  --progress  --color --display-error-details --bail",
+    "serve:dev:legacy": "node --max-old-space-size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --env.legacy=true --mode development --config ./webpack.config.js --hot --inline --progress --watch --open",
+    "serve:prod:legacy": "webpack-dev-server --env.NODE_ENV=production --env.legacy=true --mode production --config ./webpack.config.js --port 3000 --host 0.0.0.0 --hot --inline --progress  --watch --open --content-base dist/",
+    "start:legacy": "npm run serve:dev:legacy"
+  },
+  "author": "Infragistics",
+  "dependencies": {
+    "babel-runtime": "^6.26.0",
+    "core-js": "^3.6.5",
+    "igniteui-webcomponents": "^2.2.0",
```
Comment (igdmdimitrov) [inline samples/inputs/mask-input/value-modes/package.json:24]:
```suggestion
    "igniteui-webcomponents": "^3.2.0",
```

File: samples/inputs/mask-input/applying-mask/package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "build:prod:legacy": "webpack --env.NODE_ENV=production --env.legacy=true  --mode production --config ./webpack.config.js  --progress  --color --display-error-details --bail",
+    "serve:dev:legacy": "node --max-old-space-size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --env.legacy=true --mode development --config ./webpack.config.js --hot --inline --progress --watch --open",
+    "serve:prod:legacy": "webpack-dev-server --env.NODE_ENV=production --env.legacy=true --mode production --config ./webpack.config.js --port 3000 --host 0.0.0.0 --hot --inline --progress  --watch --open --content-base dist/",
+    "start:legacy": "npm run serve:dev:legacy"
+  },
+  "author": "Infragistics",
+  "dependencies": {
+    "babel-runtime": "^6.26.0",
+    "core-js": "^3.6.5",
+    "igniteui-webcomponents": "^2.2.0",
```
Comment (igdmdimitrov) [inline samples/inputs/mask-input/applying-mask/package.json:24]:
```suggestion
    "igniteui-webcomponents": "^3.2.0",
```

**Comment by igdmdimitrov** [comment]
Update `igniteui-webcomponents` version in `browser/package.json`

File: samples/inputs/mask-input/applying-mask/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v5.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample center">
+        <igc-mask-input id="mask-input" mask="(####) 00-00-00 Ext. 9999">
+          <igc-icon name="github" slot="prefix"></igc-icon>
+          <igc-icon name="github" slot="suffix"></igc-icon>
+          <span slot="helper-text">This is some helper text</span>
```
Comment (igdmdimitrov) [inline samples/inputs/mask-input/applying-mask/index.html:19]: See comment in the topic's PR.

File: samples/inputs/mask-input/value-modes/index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v5.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample center">
+        <igc-mask-input id="mask-input">
+          <igc-icon name="github" slot="prefix"></igc-icon>
+          <igc-icon name="github" slot="suffix"></igc-icon>
+          <span slot="helper-text">This is some helper text</span>
```
Comment (igdmdimitrov) [inline samples/inputs/mask-input/value-modes/index.html:19]: Change prefix icon to something `text-related` like this https://www.svgrepo.com/show/4731/text-file.svg . Remove suffix.

File: samples/inputs/mask-input/value-modes/index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v5.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample center">
+        <igc-mask-input id="mask-input">
+          <igc-icon name="file" slot="prefix"></igc-icon>
+          <span slot="helper-text">This is some helper text</span>
```
Comment (igdmdimitrov) [inline samples/inputs/mask-input/value-modes/index.html:18]:
```suggestion
```

**Review by igdmdimitrov — APPROVED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**